src/app.rs

```diff
@@ -1,10 +1,17 @@
 use clap::{crate_version, App, AppSettings, Arg};
 
 pub fn build_app() -> App<'static, 'static> {
+    let clap_color_setting =
+        if atty::is(atty::Stream::Stdout) && std::env::var_os("NO_COLOR").is_none() {
```

Comment:
I think `clap` does an `isatty` check internally, because `fd --help | cat` is not colored at the moment. So I guess we could remove the `atty::is(…)` part here (as well as the build dependency)

Reply:
isatty check removed (and rebased).